public/app/features/logs/legacyLogsFrame.ts

```diff
@@ -39,15 +46,30 @@ export function parseLegacyLogsFrame(frame: DataFrame): LogsFrame | null {
   const severityField = cache.getFieldByName('level') ?? null;
   const idField = cache.getFieldByName('id') ?? null;
 
-  const labels = getLabels(frame, cache, bodyField);
+  // extracting the labels is done very differently for old-loki-style and simple-style
+  // dataframes, so it's a little awkward to handle it,
+  // we both need to on-demand extract the labels, and also get teh labelsField,
+  // but only if the labelsField is used.
+  const [labelsField, getL] = makeLabelsGetter(cache, bodyField, frame);
+
+  const unusedFields = cache.fields.filter(
```

Comment:
Is `unusedFields` an standard property or can we consider an alternative name like `unrecognizedFields` or `unidentifiedFields`?

Reply:
@matyax 
hmmm.. it is not a standard property name no. i just named it because it is not "used" by `logsFrame.ts`... i'll think about a new name.. my only problem with `unrecognized` or `unidentified` is that they may suggest that there is an "error", but that's not the case, it is ok to have additional fields... hmm..
- `otherFields` ?
- `remainingFields` ?
- `extraFields` ?
WDYT?

Reply:
Oh, got it. I think `extraFields` is a good candidate. Feel free to choose another if you think it's better good fit.

Reply:
thanks, changed to `extraFields`